src/EVM/Expr.hs

```diff
@@ -708,7 +718,7 @@ pattern MappingSlot idx key = Keccak (WriteWord (Lit 0) key (ConcreteBuf idx))
 -- storage slots for arrays are determined by (keccak(bytes32(id)) + offset)
 -- note that `normArgs` puts the Lit as the 2nd argument to `Add`
```

Comment:
Isn't this comment outdated? `Lit` should be the first argument, no? Not the second.

Reply:
Oh wow, thanks for spotting this! Yes, definitely wrong. Fixing.